source/common/res/features/reports/incomeVsExpense/main.js

```diff
@@ -81,11 +93,27 @@
         let formattedDate = ynabToolKit.reports.formatTransactionDatel8n(transaction);
         let dateIndex = dateLabels.indexOf(formattedDate);
         let payeeId = transaction.get('payeeId');
-        let payeeData = payeesObject[payeeId];
+        let internalPayee = payeeViewModel.getPayeeById(payeeId);
+
+        // if there is no payeeId, check to see if there is a parentEntity because we might just be a subCat
+        // if there is no parent, then we're just going to have to throw this guy into null category
+        // and handle it as an unknown payee later...
+        if (!payeeId) {
+          let parentTransactionId = transaction.getParentEntityId();
+          let parentTransaction = ynab.YNABSharedLib.getBudgetViewModel_AllAccountTransactionsViewModel()
+                                      ._result.get('transactionsCollection')
```

Comment:
we probably use the promise version of it. this function doesn't have the liberty of being asynchronous so I went with this method. I'm sure there's another controller I can find with `containerLookup` but by now, the _result will most definitely be cached for us.